suite-native/app/package.json

```diff
@@ -102,6 +101,7 @@
         "expo-status-bar": "^2.0.0",
         "expo-system-ui": "^4.0.2",
         "expo-updates": "0.26.6",
+        "expo-video": "~2.0.1",
```

Comment:
I would prefer fixed version or or `^`, tilda is too benevolent for me :D

Reply:
Is not it the other way? I though that tilda is stricter than caret. https://medium.com/@codejuggler/understanding-the-difference-between-tilde-and-caret-in-package-json-aacb092a67b7

Reply:
Tilda is stricter, but we have fixed versions thanks to yarn.lock anyway... But mixing those two brings even more confusion?

Reply:
Alright, I unified so the caret is used (same for all the expo packages).